python.d/pip_install.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+# FIXME
+# 1. make sure you are in the proper directory
+# 2. In fedora 24, the python modules names are not known
+
 PIP=`which pip`
 
 ${PIP} install --target="python_modules" yaml
```

Comment:
@1 It just needs to check if there is `python_modules` directory and even if there isn't, it won't work. But you can add something like this after shebang:
```sh
[ -d 'python_modules' ] || ( echo "Cannot find 'python_modules' directory, exiting"; exit 1)
```

@2 In Python 3 packages are named differently, and there is no `mysqldb` in pypi repository.

```sh
${PIP} install --target="python_modules" yaml || ${PIP} install --target="python_modules" PyYAML
${PIP} install --target="python_modules" mysqldb || ${PIP} install --target="python_modules" pymysql
```